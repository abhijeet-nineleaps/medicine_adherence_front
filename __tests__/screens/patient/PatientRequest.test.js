import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import {shallow} from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Patientrequest from '../../../src/screens/patient/PatientRequest';
Enzyme.configure({adapter: new Adapter()});
jest.mock('@react-navigation/native', () => ({
  ...jest.requireActual('@react-navigation/native'),
  useFocusEffect: jest.fn(),
}));
describe('Click send image', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Patientrequest />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  // it('test open save button', () => {
  //   const mockFn = jest.fn();
  //   const wrapper = shallow(<Patientrequest acceptrequest={mockFn} />);
  //   wrapper.find('#accept').simulate('press');
  // });
  // it('test open save button', () => {
  //   const mockFn = jest.fn();
  //   const wrapper = shallow(<Patientrequest deletereq={mockFn} />);
  //   wrapper.find('#delete').simulate('press');
  // });
});
