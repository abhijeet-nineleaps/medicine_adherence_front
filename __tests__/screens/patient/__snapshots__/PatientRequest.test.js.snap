// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Click send image renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <RCTScrollView
    data={Array []}
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    refreshControl={
      <RefreshControlMock
        onRefresh={[Function]}
        refreshing={false}
      />
    }
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    viewabilityConfigCallbackPairs={Array []}
  >
    <RCTRefreshControl />
    <View />
  </RCTScrollView>
  <View
    style={
      Object {
        "alignSelf": "center",
        "position": "absolute",
        "top": 90,
      }
    }
  >
    <Image
      resizeMode="contain"
      source={
        Object {
          "testUri": "../../../assests/images/nopatientreq.png",
        }
      }
      style={
        Object {
          "width": 250,
        }
      }
    />
  </View>
</View>
`;
